wordpress-mobile/WordPress-iOS
# Gutenberg - Add list block v2 flag

**Related PRs:**
- **Gutenberg Mobile PR**: https://github.com/wordpress-mobile/gutenberg-mobile/pull/5054
- **Gutenberg PR**: https://github.com/WordPress/gutenberg/pull/42702
- **WordPressKit iOS PR**: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/535

Adds a Gutenberg feature flag for the List V2 block.

Fixes:
- https://github.com/WordPress/gutenberg/issues/42624

To test:
Check the [Gutenberg PR description](https://github.com/WordPress/gutenberg/pull/42702) for further testing instructions.

## Regression Notes
1. Potential unintended areas of impact
Block editor

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual block editor testing

4. What automated tests I added (or what prevented me from doing so)
None. This feature flag is temporary until List v2 is fully implemented by web and mobile.

PR submission checklist:
